src/components/NotificationsAlert.tsx

```diff
@@ -22,6 +22,7 @@ const Wrapper = styled.div`
 
 export const NotificationsAlert = () => {
   const signedIn = useAuthStore((store) => store.signedIn);
+  const isSignedIn = useMemo(() => signedIn, [signedIn]);
```

Comment:
I believe this usage of `useMemo()` isn't really achieving anything. Wouldn't it be the same as using `signedIn` directly?